### ISSUE TITLE
{Freeway-Bug}[SampleCard-Agenda; Config-windows-timeline.json]: The color contrast ratio for graphical component 'LocationC image', 'CalendarConflict image', 'TravelByCar image' is not sufficient.

### ISSUE THREAD

**Issue by Bhaskar301101** [microsoft/AdaptiveCards#8473]
### Target Platforms

Android

### SDK Version

 2023.03.31.1

### Application Name

Adaptive Card

### Problem Description

Test Environment:
OS: Android 12
Device: S21+
Screen reader: Talkback
AdaptiveCard Android App version:  2023.03.31.1

Pre-requisite:
Import-Sample Cards->Agenda.json; Config-windows-timeline

Repro Steps:

1. Navigate to the above sample card.
2. Observe the contrast issue for graphical components like 'LocationC image', 'CalendarConflict image', 'TravelByCar image'

Actual Result:
The color contrast ratio for graphical component 
'LocationC image': CCA is 1.25:1
'CalendarConflict image': CCA is 1.28:1
'TravelByCar image': CCA is 2.41:1

Expected Result:
The color contrast ratio for graphical components should be atleast 3:1.

User Experience:
Low vision users will get impact if color contrast ratio for meaningful images does not meet the requirement.

WCAG Reference:
https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast

Note:
This issue is repro for below configs,
windows-notification.json: 'LocationC image', 'CalendarConflict image' repro this issue.
Webchat.json: 'TravelByCar image', 'LocationACoffee image' repro this issue.
testVariantHostConfig.json: 'LocationC image', 'CalendarConflict image' repro this issue.
skype.json: 'TravelByCar image', 'LocationACoffee image' repro this issue.
Sample.json: 'TravelByCar image', 'LocationACoffee image' repro this issue.
Render-to-image.json: 'TravelByCar image', 'LocationACoffee image' repro this issue.
Outlook-desktop.json: 'TravelByCar image', 'LocationACoffee image' repro this issue.
microsoft-teams-light.json: 'TravelByCar image', 'LocationACoffee image' repro this issue.
microsoft-teams-dark.json: 'LocationC image', 'CalendarConflict image' repro this issue.
cortana-skills-light:  'TravelByCar image', 'LocationACoffee image' repro this issue.
cortana-skills-dark:  'LocationC image', 'CalendarConflict image' repro this issue.

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “,

### Screenshots

![Low CCA for LocationC image](https://user-images.githubusercontent.com/89897257/232739563-014cbbf0-e57b-442a-9a86-4660d86bd948.jpg)
![Low CCA for calendar conflict](https://user-images.githubusercontent.com/89897257/232739580-4d165f25-1b97-4115-9a52-f725c7b92054.jpg)
![Low CCA for TravelBycar image](https://user-images.githubusercontent.com/89897257/232739585-c571c22e-6193-4600-8d21-f689d66633cc.jpg)

### Card JSON

```JSON
NA
```

**Comment by jwoo-msft** [comment]
one option: disable host configuration
2nd option: change background color

**Comment by vagpt** [comment]
This issue is not repro on the environment below as there are some UI change, hence closing this issue.

**Test Environment:**
OS: Android 13
Device: S20
Screen Reader: Talkback
AdaptiveCard Android App version: 2023.12.12.1 (85454887)

![image](https://github.com/microsoft/AdaptiveCards/assets/95913748/65d81652-0658-4e13-ba0c-808dd1e779e1)
